### PR TITLE
More combat/gun/medical tweaks.

### PR DESCRIPTION
**Pull request by NonQueueingMatt**
Combat is generally a bit too non-lethal right now. In the words of many, people don't really die when they should and it's far too easy to limp back to medical, get healed and go back in the fray instantly. So, here's what I did to fix it.

First of all, the bugfixes:
- Fixed pointblank messages.
- Fixed hearts displaying a positive pulse after the person is dead.

And now, the big bad part, the changes:
 - Blue grabs no longer make the grabbed person drop their held things.
 - Choking people now starts a timer, at the end of which the afflicted person will go unconscious.
 - Scarring can now go up to reducing an organ's maximum health by 50%.
 - Brain scarring can now happen while below 60% blood oxygenation.
 - Brain health from low oxygenation should decay faster. Both because the last two levels do more damage overall and also because the thresholds at which the past two levels stop doing damage have been reduced.
 - Brute weapons now inflict slightly more pain than before.
 - Tweaked organ spillover damage code. This will probably have some effects on how damage is calculated, as the old one was not updated.

In general, this makes things hurt more and makes people live for less after receiving grievous wounds. The spillover code is ported from Bay as I forgot to port it during the original brainmed PR. Oops.

### CODE REVIEW

**Comment by NonQueueingMatt** [comment]
!review

File: code/modules/organs/organ_external.dm [hunk 1]
```diff
@@ -250,15 +251,52 @@
...
+			//need to check sharp again here so that blunt damage that was strong enough to break skin doesn't give puncture wounds
+			if(sharp && !edge)
+				to_create = PIERCE
+		createwound(to_create, brute)
+
+	if(burn)
+		if(laser)
+			createwound(LASER, burn)
+		else
+			world << "burn"
```
[the remark this answers is not in the document]
Comment (NonQueueingMatt) [reply code/modules/organs/organ_external.dm:291]: bruh

File: code/modules/mob/mob_grab.dm [hunk 2]
```diff
@@ -159,9 +159,16 @@
...
 			var/mob/living/carbon/human/A = affecting
+			var/obj/item/clothing/C = A.head
+			if(C && (C.item_flags & THICKMATERIAL))
+				return
 			if(!(A.species.flags & NO_BREATHE))
 				A.losebreath = max(A.losebreath + 3, 5)
 				A.adjustOxyLoss(3)
+				if(affecting.stat == CONSCIOUS)
+					if(do_mob(assailant, affecting, 150))
+						A.visible_message(SPAN_WARNING("[A] falls unconscious..."), SPAN_DANGER("The world goes dark as you fall unconscious..."))
```
Comment (Geevies) [inline code/modules/mob/mob_grab.dm:170]:
```suggestion
						A.visible_message(SPAN_WARNING("[A] falls unconscious..."), FONT_LARGE(SPAN_DANGER("The world goes dark as you fall unconscious...")))
```

Comment (Geevies) [reply code/modules/mob/mob_grab.dm:170]: should make it easier to see why you're paralyzed during combat

File: code/modules/organs/internal/_internal.dm [hunk 3]
```diff
@@ -43,11 +43,11 @@
...
 /obj/item/organ/internal/proc/surgical_fix(mob/user)
 	if(damage > min_broken_damage)
-		var/scarring = damage/max_damage
-		scarring = 1 - 0.3 * scarring ** 2 // Between ~15 and 30 percent loss
+		var/scarring = damage / max_damage
+		scarring = 1 - 0.5 * scarring ** 2 // Between ~15 and 50 percent loss.
 		var/new_max_dam = Floor(scarring * max_damage)
 		if(new_max_dam < max_damage)
-			to_chat(user, "<span class='warning'>Not every part of [src] could be saved, some dead tissue had to be removed, making it more suspectable to damage in the future.</span>")
+			to_chat(user, SPAN_WARNING("Not every part of [src] could be saved, some dead tissue had to be removed, making it more susceptible to damage in the future."))
```
Comment (Doxxmedearly) [inline code/modules/organs/internal/_internal.dm:50]:
```suggestion
			to_chat(user, SPAN_WARNING("Not every part of [src] could be saved; some dead tissue had to be removed, making it more susceptible to future damage."))
```
Small grammar improvement.

File: code/modules/mob/mob_grab.dm (hunk 2 above)
Comment (NonQueueingMatt) [reply code/modules/mob/mob_grab.dm:170]: You're not in combat if you got neck grabber for 30+ seconds

**Pull request merged**